tests/conf/ssl4eo_l_benchmark_cdl.yaml

```diff
@@ -0,0 +1,20 @@
+module:
+  _target_: torchgeo.trainers.SemanticSegmentationTask
+  loss: "ce"
+  model: "unet"
+  backbone: "resnet18"
+  weights: null
+  learning_rate: 1e-3
+  learning_rate_schedule_patience: 6
+  in_channels: 7
+  num_classes: 17
```

Comment:
CDL actually has far more classes than this, although it doesn't matter too much for the tests